addons/website_event_track/static/src/js/website_event_track_proposal.js

```diff
@@ -0,0 +1,182 @@
+odoo.define('website_event_track.website_event_track_proposal', function (require) {
+    'use strict';
+
+    var core = require('web.core');
+    var publicWidget = require('web.public.widget');
+    var utils=require('web.utils');
+
+    var _t = core._t;
+
+    publicWidget.registry.websiteEventProposalForm = publicWidget.Widget.extend({
```

Comment:
Some generic guidelines remarks here ! 

* put handlers before private methods, with a clear separator (public / handlers / private);
* order methods inside a given section alphabetically;
* try to improve naming of methods (_onNameInputPropagateOnContact is a bit too much details and _send lacks details);